.gitattributes

```diff
@@ -0,0 +1,15 @@
+# Lock batch file ending
+*.bat text eol=crlf
```

Comment:
Based on my reading, this line says...

    Unless I say otherwise, consider all .bat files anywhere in
    the project to be text, and force their line endings to be crlf